### PR TITLE
Update to latest version of gradle, add gradle wrapper task

### PR DESCRIPTION
**Pull request by barnhill**
Gradle version used to build this project is extremely hold.  This will update to the latest at the time this PR is opened along with creating a very common gradle task for the gradle wrapper.  Updating the gradle version from this PR forward just consists of updating the version in the build.gradle and running `./gradlew wrapper` twice.  This will update the gradle.properties, and subsequent auto generated gradle files/scripts.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,3 +1,7 @@
+wrapper {
+    gradleVersion = '6.8.3'
+    distributionType = Wrapper.DistributionType.BIN
+}
```
Comment (barnhill) [inline build.gradle:4]: This code allows for specifying the gradle wrapper version in the build gradle and updating all the generated code by running `./gradlew wrapper` twice

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,5 @@
-#Thu Oct 26 11:42:11 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
 zipStoreBase=GRADLE_USER_HOME
```
Comment (barnhill) [inline gradle/wrapper/gradle-wrapper.properties:4]: auto updated with the gradle wrapper task

File: gradlew [hunk 3]
```diff
@@ -1,25 +1,59 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```
Comment (barnhill) [inline gradlew:1]: auto updated with the gradle wrapper task

File: gradlew.bat [hunk 4]
```diff
@@ -1,3 +1,19 @@
+@rem
```
Comment (barnhill) [inline gradlew.bat:1]: auto updated with the gradle wrapper task

File: build.gradle (hunk 1 above)
Comment (barnhill) [reply build.gradle:4]: All other files in this PR are automatically generated/updated using the gradle wrapper task

**Review by benjamin-bader — APPROVED** [review]

**Comment by barnhill** [comment]
Ready to merge this?

**Comment by benjamin-bader** [comment]
I don't have merge permissions anymore.  @emarc-m would you please take a look and merge if you think it's ready?

**Pull request merged**